cognite/neat/rules/exporter/_rules2dms.py

```diff
@@ -292,82 +326,109 @@ def views_from_rules(rule: Rules, space: str | None = None) -> dict[str, ViewApp
-                    description=property_definition.description,
-                )
-            else:
-                logging.warning(exceptions.ViewPropertyTypeUnsupported(property_id).message)
-                warnings.warn(
-                    exceptions.ViewPropertyTypeUnsupported(property_id).message,
-                    category=exceptions.ViewPropertyTypeUnsupported,
-                    stacklevel=2,
-                )
+        if errors:
+            raise ExceptionGroup("View properties have been redefined! This is prohibited! Aborting!", errors)
+
+        return views
+
+    @staticmethod
+    def view_property_from_rules_property(
+        property_: Property, space: str
+    ) -> MappedPropertyApply | ConnectionDefinitionApply | None:
+        property_.container = cast(Container, property_.container)
+        property_.container_property = cast(str, property_.container_property)
+        if property_.property_type == "DatatypeProperty":
+            return MappedPropertyApply(
+                container=ContainerId(space=property_.container.space, external_id=property_.container.external_id),
+                container_property_identifier=property_.container_property,
+                name=property_.property_name,
+                description=property_.description,
+            )
+
+        # edge 1-1 == directRelation
+        elif property_.property_type == "ObjectProperty" and property_.max_count == 1:
+            return MappedPropertyApply(
+                container=ContainerId(space=property_.container.space, external_id=property_.container.external_id),
+                container_property_identifier=property_.container_property,
+                name=property_.property_name,
+                description=property_.description,
+                source=ViewId(
+                    space=property_.expected_value_type.space,
+                    external_id=property_.expected_value_type.external_id,
+                    version=property_.expected_value_type.version,
+                ),
+            )
 
-        return view_properties
+        # edge 1-many == EDGE, this does not have container! type is here source ViewId ?!
+        elif property_.property_type == "ObjectProperty" and property_.max_count != 1:
+            return SingleHopConnectionDefinitionApply(
+                type=DirectRelationReference(space=space, external_id=f"{property_.class_id}.{property_.property_id}"),
+                # Here we create ViewID to the container that the edge is pointing to.
+                source=ViewId(
+                    space=property_.expected_value_type.space,
+                    external_id=property_.expected_value_type.external_id,
+                    version=property_.expected_value_type.version,
+                ),
+                direction="outwards",
+                name=property_.property_name,
+                description=property_.description,
+            )
+        else:
```

Comment:
Not for this PR, but note that we now have three more Connection types:
![image](https://github.com/cognitedata/neat/assets/60234212/af051943-d3ff-4db2-a07b-46d0389f7df9)

There are trade-offs for edges vs direct relations:

https://docs.cognite.com/cdf/dm/dm_concepts/dm_spaces_instances#direct-relations-vs-edges

Reply:
We need examples for each, and to devise condition in Rules file when each occur. 
Currently it is simple:
- as long as it is not listable object property, then it is resolved as direction relation
- otherwise it is edge